src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final String FORM = "Figure: circle, area: %.1f sq.units, radius: %d units"
+            + ", color: %s";
```

Comment:
You don't need to store this string as a constant, just write it in toString(). Fix everywhere

Reply:
>  (you shouldn't override toString() method for this)

 So I moved them to draw() methods.